tests/FakeItEasy.Tests/ACallToTests.cs

```diff
@@ -50,5 +68,22 @@ public void CallTo_with_function_call_should_return_configuration_from_configura
             // Assert
             result.Should().BeSameAs(configuration);
         }
+
+        [SuppressMessage("Microsoft.Design", "CA1062:Validate arguments of public methods", MessageId = "0", Justification = "It's not really a public method")]

```

Comment:
I'm surprised we're not hounded by this in all the specs. I don't see a suppression.

I could support this moving to the global suppression file for all test projects, if you'd like me to add it to my PR.